### DISCUSSION TITLE
Saleor 3.0 Stable Release is OUT!

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#9136] [Show and tell]
The long-awaited stable release of Saleor 3.0 is now available! 

It's the biggest Saleor update to date. It introduces multichannel, powerful extensibility, and MUCH more! We’d like to thank you for your patience, contribution, and support.

You can find the changelog here: https://github.com/saleor/saleor/releases/tag/3.0.0

P.S. A blog post summarizing the most important features will be released next week. Stay tuned!

- [Twitter](https://twitter.com/getsaleor/status/1491759466982825988)

**Comment by AbduRahmanS** [comment]
Hi, I started with saleor 2 days ago, i'm testing it in local  with saleor-platform. 
Very happy to come accross this new stable release, But is saleor platform ready for it ?

**Reply by karolkielecki** [reply]
The saleor-platform is not ready yet. There are also some other issues related to that repo reported by some of our community members.
